cloudera/dbt-spark-cde
# [DBT-488] add instrumentation for connection parameters

## Describe your changes
add instrumentation for connection parameters

## Internal Jira ticket number or external issue link
DBT-488 (Internal)

## Testing procedure/screenshots(if appropriate):
For a valid dbt-spark-cde profile, run dbt debug, the logs/dbt.log should have something as follows:

5:34:08.421353 [debug] [Thread-1  ]: Tracker adapter: Sending Event [{"data": {"event_type": "dbt_spark_cde_open", "auth": "cde", "connection_state": "open", "elapsed_time": "1.79", "incremental_strategy": "N/A", "model_name": "N/A", "model_type": "N/A", "permissions": "N/A", "profile_name": "N/A", "sql_type": "N/A"}}]

## Checklist before requesting a review
